modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -129,6 +129,146 @@ G_TYPED_KERNEL(GBackgroundSubtractor, <GMat(GMat, BackgroundSubtractorParams)>,
     }
 };
 
+/** @brief Structure for the Kalman filter's initialization parameters.*/
+
+struct GAPI_EXPORTS KalmanParams
+{
+    //! Type of the created matrices that should be CV_32F or CV_64F.
+    int type = CV_32FC1;
```

Comment:
I think this `type` field should be also removed since we have it in all the matrices and there is no need to set it separately

Reply:
Done